frontend/components.js

```diff
@@ -62,11 +62,18 @@ const WalletHeader = (state) => `
   </div>
 `
 
+const address = (address, isTransaction = false) => `
+          <div class="address-wrap">
+          <input readonly type="text" class="address" value="${address}"/>
+          <a href="https://cardanoexplorer.com/${isTransaction ? 'tx' : 'address'}/${address}" target="_blank" title="examine via CardanoExplorer.com"></a>
```

Comment:
It would be nice to have there also a button to copy the address to clipboard. But at the same time I'm not really sure if it is a good idea for (smaller) touch screens. Maybe @darjanin could give his opinion/suggestions

Reply:
What about option when the input's focus event is fired, the content of the input is copied to clipboard? We will show some tooltip about it later saying what happened. 
If it has some issues, like you like to select your address without copying, then I would put there two buttons - one is existing link, next will be <kbd>Copy to 📋</kbd> that will be on the end of the address input, and on the mobile this two button will be below address bar. 
 
```
# desktop version
|.address----------------| Copy to 📋 | 🔗 |

# mobile version
|.address----------------|
|--copy to 📋--|---🔗---|
```

Reply:
I prefer the second option, i.e. two buttons. I generally don't like when the clipboard gets overwritten without prior notice.

Reply:
but for aesthetics: imho choose text or icon, not both, I prefer icons, i they are really intuitive (e.g. 📋 is for me ok, but  🔗 is not)

Reply:
I just used emojis for communicating the idea. At the moment icon is fine. For copy to clibpoard can be used same icon as Github uses.